src/commands/suggest.ts

```diff
@@ -0,0 +1,47 @@
+// Codey suggest Command
+import Discord from 'discord.js';
+import { openDB } from '../components/db';
+
+/*
+  // Keeping this here just to keep my debugging tools
+  db.run('DROP TABLE suggestions');
+  const res = await db.all('SELECT * FROM suggestions');
+  for (const rows of res) {
+    console.log(rows['id'], rows['created_at'], rows['author'], rows['suggestion'], rows['state']);
+  }
+*/
+
+// All states of suggestion records
+enum SuggestionStates {
+  Create = 'Create',
```

Comment:
nit: rename to `Created` makes more sense? Also since we are using enums, the actual values could be integers.
```
enum SuggestionStates {
Created = 1,
Rejected,
Accepted,
Pending
}
```
Sorry if I was kinda misleading you, I wanted you to use the full name in the DB because there wasn't enough reference in the code to understand what the states meant. Now that we have this enum, we can just reference the it for more context, so we could just store integers in the DB.